dotcom-rendering/src/web/components/Carousel.importable.tsx

```diff
@@ -437,14 +437,23 @@ const HeaderAndNav: React.FC<HeaderAndNavProps> = ({
 const getImageUrl = (source: string | undefined) => {
 	if (!source) return undefined;
 	const url = new URL(source);
-	if (
-		url.hostname === 'i.guim.co.uk' &&
-		url.pathname.startsWith('/img/media/')
-	) {
-		return `https://media.guim.co.uk/${url.pathname.replace(
-			'/img/media/',
-			'',
-		)}`;
+
+	const buckets = ['/img/media/', '/img/static/'] as const;
+	const prefix = buckets.find((bucket) => url.pathname.startsWith(bucket));
+
+	if (url.hostname === 'i.guim.co.uk' && prefix) {
+		switch (prefix) {
+			case '/img/media/':
+				return `https://media.guim.co.uk/${url.pathname.replace(
+					prefix,
+					'',
+				)}`;
+			case '/img/static/':
+				return `https://static.guim.co.uk/${url.pathname.replace(
```

Comment:
Why are we changing the hostname here? I'm guessing if we do then fastly resizer wouldn't do anything?

Reply:
This is where the master files are hosted. It’s being turned back into a `i.guim` domain by `Picture.tsx`:

https://github.com/guardian/dotcom-rendering/blob/6e143dbc2a63d0c9d884427f1b47d737d6c00cde/dotcom-rendering/src/web/components/Picture.tsx#L180-L181

This whole approach is a temporary reverse-engineering of the signing process.

Reply:
Ah okay ... definitely weird, we should get the full solution done asap then!